src/main/java/org/ohdsi/authenticator/service/rest/RestAuthService.java

```diff
@@ -15,22 +14,13 @@
 import org.springframework.expression.ExpressionParser;
 import org.springframework.expression.spel.standard.SpelExpressionParser;
 import org.springframework.expression.spel.support.StandardEvaluationContext;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpMethod;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.*;
```

Comment:
Looks like different intellij settings, @wivern do you want to discuss wildcard import settings?

Reply:
done